### PR TITLE
Fixed #24208 -- Documented changes in private model relations.

### PR DESCRIPTION
**Pull request by timgraham**

### CODE REVIEW

**Comment by PirosB3** [comment]
Nice!

**Comment by collinanderson** [comment]
Yes. Very thorough.

File: docs/releases/1.8.txt [hunk 1]
```diff
@@ -915,6 +915,44 @@ Also private APIs ``django.template.base.compile_string()``,
 ``django.template.loader.find_template()``, and
 ``django.template.loader.get_template_from_string()`` were removed.
 
+``model`` attribute on private model relations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In earlier versions of Django, on a model with a reverse foreign key
+relationship (for example), ``model._meta.get_all_related_objects()`` returns
```
Comment (MarkusH) [inline docs/releases/1.8.txt]: I'd use past tense here: "returns" --> "returned" at the end of the line.

**Comment by MarkusH** [comment]
LGTM

**Comment by PirosB3** [comment]
LGTM

**Pull request merged**